Sources/SwiftTemplate.swift

```diff
@@ -3,9 +3,16 @@
 // Copyright (c) 2017 SwiftGen
 // MIT Licence
 //
-
+import Foundation
 import Stencil
 
+#if os(Linux)
+#if swift(>=3.1)
```

Comment:
Indeed `#if swift(<3.1)` isn't a valid syntax… But I think `#if ! swift(>=3.1)` is. This would avoid an empty `#if` body and the main logic in the `#else`

You could even probably just write `#if os(Linux) && ! swift(>=3.1)`